tools/clang/test/CodeGenSPIRV_Lit/spirv.legal.sbuffer.struct.hlsl

```diff
@@ -49,59 +49,59 @@ float4 foo(Combine comb);
 float4 main() : SV_Target {
     Combine c;
 
-// CHECK:      [[ptr:%\d+]] = OpAccessChain %_ptr_Function__ptr_Uniform_type_AppendStructuredBuffer_v4float %c %int_0 %int_0
+// CHECK:      [[ptr:%[0-9]+]] = OpAccessChain %_ptr_Function__ptr_Uniform_type_AppendStructuredBuffer_v4float %c %int_0 %int_0
 // CHECK-NEXT:                OpStore [[ptr]] %gASBuffer
     c.s.append = gASBuffer;
-// CHECK:      [[ptr:%\d+]] = OpAccessChain %_ptr_Function__ptr_Uniform_type_ConsumeStructuredBuffer_v4float %c %int_0 %int_1
-// CHECK-NEXT:                OpStore [[ptr]] %gCSBuffer
+// CHECK:      [[ptr_0:%[0-9]+]] = OpAccessChain %_ptr_Function__ptr_Uniform_type_ConsumeStructuredBuffer_v4float %c %int_0 %int_1
+// CHECK-NEXT:                OpStore [[ptr_0]] %gCSBuffer
     c.s.consume = gCSBuffer;
 
     T t;
-// CHECK:      [[ptr:%\d+]] = OpAccessChain %_ptr_Function__ptr_Uniform_type_StructuredBuffer_Basic %t %int_0
-// CHECK-NEXT:                OpStore [[ptr]] %gSBuffer
+// CHECK:      [[ptr_0_0:%[0-9]+]] = OpAccessChain %_ptr_Function__ptr_Uniform_type_StructuredBuffer_Basic %t %int_0
+// CHECK-NEXT:                OpStore [[ptr_0_0]] %gSBuffer
     t.ro = gSBuffer;
-// CHECK:      [[ptr:%\d+]] = OpAccessChain %_ptr_Function__ptr_Uniform_type_RWStructuredBuffer_Basic %t %int_1
-// CHECK-NEXT:                OpStore [[ptr]] %gRWSBuffer
+// CHECK:      [[ptr_0_0_0:%[0-9]+]] = OpAccessChain %_ptr_Function__ptr_Uniform_type_RWStructuredBuffer_Basic %t %int_1
+// CHECK-NEXT:                OpStore [[ptr_0_0_0]] %gRWSBuffer
     t.rw = gRWSBuffer;
-// CHECK:      [[val:%\d+]] = OpLoad %T %t
-// CHECK-NEXT: [[ptr:%\d+]] = OpAccessChain %_ptr_Function_T %c %int_1
-// CHECK-NEXT:                OpStore [[ptr]] [[val]]
+// CHECK:      [[val:%[0-9]+]] = OpLoad %T %t
+// CHECK-NEXT: [[ptr_0_0_0_0:%[0-9]+]] = OpAccessChain %_ptr_Function_T %c %int_1
+// CHECK-NEXT:                OpStore [[ptr_0_0_0_0]] [[val]]
     c.t = t;
 
-// CHECK:      [[ptr:%\d+]] = OpAccessChain %_ptr_Function__ptr_Uniform_type_ByteAddressBuffer %c %int_2
-// CHECK-NEXT:                OpStore [[ptr]] %gBABuffer
+// CHECK:      [[ptr_0_0_0_0_0:%[0-9]+]] = OpAccessChain %_ptr_Function__ptr_Uniform_type_ByteAddressBuffer %c %int_2
+// CHECK-NEXT:                OpStore [[ptr_0_0_0_0_0]] %gBABuffer
     c.ro = gBABuffer;
-// CHECK:      [[ptr:%\d+]] = OpAccessChain %_ptr_Function__ptr_Uniform_type_RWByteAddressBuffer %c %int_3
-// CHECK-NEXT:                OpStore [[ptr]] %gRWBABuffer
+// CHECK:      [[ptr_0_0_0_0_0_0:%[0-9]+]] = OpAccessChain %_ptr_Function__ptr_Uniform_type_RWByteAddressBuffer %c %int_3
+// CHECK-NEXT:                OpStore [[ptr_0_0_0_0_0_0]] %gRWBABuffer
```

Comment:
Could we have better numbering for this? `prt_0`, `ptr_1`, .... It is hard to count the number of zeros and match them up.

Reply:
Fixed.